packages/website/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "endOfLine": "auto"
+}
```

Comment:
As mentioned above, this is already handled via ESLint. There's no need to call or use prettier separately.

Reply:
The prettier and prettier-eslint plugins were both unfortunately unable to detect styling without it and would error out without it. The alternative is to do the same but in the eslint file or for future editors to also be unable to autolint